code/game/objects/items/reagent_containers/hypospray.dm

```diff
@@ -97,11 +97,11 @@
 		overlays += filling
 
 
-
 /obj/item/reagent_container/hypospray/tricordrazine/New()
 	..()
-	reagents.add_reagent("tricordrazine", 60)
+	list_reagents = list("tricordrazine" = 60)
 
 /obj/item/reagent_container/hypospray/oxycodone/New()
 	..()
-	reagents.add_reagent("oxycodone", 60)
+	list_reagents = list("oxycodone", 60)
```

Comment:
You have to add the reagent list to a path, or at least before `..()` for it to actually work.
Also you forgot to switch on the equal sign on the latter.